rosdep/python.yaml

```diff
@@ -6730,6 +6730,9 @@ python3-texttable:
   gentoo: [dev-python/texttable]
   rhel: ['python%{python3_pkgversion}-texttable']
   ubuntu: [python3-texttable]
+python3-thriftpy:
+  fedora: [python3-thriftpy]
+  ubuntu: [python3-thriftpy]
```

Comment:
Since this is not available on Xenial, I'll suggest:

```
ubuntu:
  '*': [python3-thiftpy]
  xenial: null
```

Reply:
I'd actually suggest using the pip rule specifically for xenial. Since it's not available there via debian package it won't conflict so we can use the pip rule as a fallback on that platform.

Reply:
That works too :+1: